examples/with-pullstate/components/dark-mode-demo.js

```diff
@@ -0,0 +1,29 @@
+import { PullstateCore } from '../stores'
+
+export default function DarkModeDemo() {
+  const { UIStore } = PullstateCore.useStores()
+  const isDarkMode = UIStore.useState((s) => s.isDarkMode)
+
+  return (
+    <div
+      style={{
+        backgroundColor: isDarkMode ? '#333' : '#CCC',
+        color: isDarkMode ? '#CCC' : '#333',
+      }}
+    >
+      <h1>Pullstate Demo</h1>
+
+      <p>Dark Mode is {isDarkMode ? 'On' : 'Off'}</p>
+
+      <button
+        onClick={() =>
+          UIStore.update((s) => {
+            s.isDarkMode = !s.isDarkMode
+          })
+        }
+      >
+        Toggle Dark Mode
+      </button>
+    </div>
```

Comment:
This component should have links to the other pages, that's important to see how all pages work together while the app is running.